supported_formats_and_operations.md

```diff
@@ -43,6 +43,12 @@ The supported preprocessing operations.
     - `min_percentile` the lower percentile used for normalization, in range 0 to 100.
     - `max_percentile` the upper percentile used for normalization, in range 0 to 100. Has to be bigger than `upper_percentile`.
   - `reference_implementaion`
+- `scale_linear` scale the tensor with a fixed multiplicative and additive factor
+  - `kwargs`
+    - `gain` multiplicative factor
+    - `offset` additive factor
+    - `axes` the subset of axes to scale jointly. For example `xy` to scale the two image axes for 2d data jointly. The batch axis (`b`) is not valid here.
```

Comment:
```suggestion
```
linear scale always applies to all axes

Reply:
Why?

Reply:
I dpn't see any reason why we wouldn't allow to scale axes separately. In this case `gain` and `offset` need to be lists of course.

Reply:
ah, of course, sorry for the confusion